src/pages/terraform/cdktf/[[...page]].tsx

```diff
@@ -1,19 +1,10 @@
-import terraformData from 'data/terraform.json'
-import { ProductData } from 'types/products'
-import { getStaticGenerationFunctions } from 'views/docs-view/server'
 import DocsView from 'views/docs-view'
+import { getRootDocsPathGenerationFunctions } from 'views/docs-view/utils/get-root-docs-path-generation-functions'
```

Comment:
Introduction of `getRootDocsPathGenerationFunctions` reduces boilerplate for all page files. Will expand this to other products in a subsequent PR, to get back to a single `views/docs-view/server.ts` file. See #750 for a start on that.